src/platform/Linux/BLEManagerImpl.cpp

```diff
@@ -222,8 +226,18 @@ void BLEManagerImpl::_OnPlatformEvent(const ChipDeviceEvent * event)
         break;
 
     case DeviceEventType::kCHIPoBLEWriteReceived:
-        HandleWriteReceived(event->CHIPoBLEWriteReceived.ConId, &CHIP_BLE_SVC_ID, &ChipUUID_CHIPoBLEChar_RX,
-                            PacketBufferHandle::Adopt(event->CHIPoBLEWriteReceived.Data));
+#if !CHIP_DEVICE_CONFIG_SUPPORTS_CONCURRENT_CONNECTION
+        // If BLE is being stopped, ignore any write messages
+        if (mState != kState_Initialized)
+        {
+            ChipLogProgress(DeviceLayer, "Non-concurrent mode BLE stopped, BLE write ignored");
+        }
+        else
+#endif
```

Comment:
It's really not clear why this is being done. Please document more detauks about how we could get here. Why is it specific to non-concurrent? This seems like always an error that could be an early return that is not conditional to the `#if !CHIP_DEVICE_CONFIG_SUPPORTS_CONCURRENT_CONNECTION`

Reply:
BlueZ sends another message after the BLE has been stopped which was causing the Wi-Fi to be disconnected and reestablished slowing down commissioning. This is a good point, I have modified the system so that BlueZ is disconnected at the same time of the underlying BLE, negating the need for this. So removed.